apps/ledger-live-mobile/src/components/Web3AppWebview/liveSDKLogic.ts

```diff
@@ -6,22 +6,15 @@ import BigNumber from "bignumber.js";
 
 export default function prepareSignTransaction(
   account: AccountLike,
-  parentAccount: Account | undefined,
-  liveTx: Partial<Transaction & { gasLimit: BigNumber }>,
+  parentAccount: Account | null | undefined,
+  liveTx: Partial<
+    Transaction & {
+      gasLimit: BigNumber;
+    }
+  >,
 ): TransactionCommon {
   const bridge = getAccountBridge(account, parentAccount);
-  const t = bridge.createTransaction(account);
```

Comment:
Is it okay to not go through the `createTransaction` anymore ?
I understand you made it the same as LLD but I think that some special defaults could be setup in this function IIRC